paper.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Matching: A Python library for solving matching games'
+tags:
+  - Python
+  - mathematics
+  - economics
+  - computer science
+  - game theory
+  - stability
+authors:
+  - name: Henry Wilde
+    orcid: 0000-0002-3788-7691
+    affiliation: 1
+  - name: Vincent Knight
+    orcid: 0000-0002-4245-0638
+    affiliation: 1
+  - name: Jonathan Gillard
+    orcid: 0000-0001-9166-298X
+    affiliation: 1
+affiliations:
+  - name: School of Mathematics, Cardiff University
+    index: 1
+date: February 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+Matching games allow for the allocation of resources and partnerships in a fair
+way. Typically, a matching game is defined by two sets of players that each have
+preferences over at least some of the elements of the other set. The objective
+of the game is then to find a mapping between the sets of players in which
+everyone is *happy enough* with their match.
+
+One of the most ubiquitous matching games models the Stable Marriage Problem
```

Comment:
`One of the most ubiquitous matching game models is the Stable Marriage Problem (SM)`